packages/reports/src/Containers/progress-slider-stream.tsx

```diff
@@ -1,20 +1,21 @@
 import React from 'react';
 import { ProgressSlider } from '@deriv/components';
 import { getCurrentTick, TContractInfo, getCardLabels } from '@deriv/shared';
-import { connect } from 'Stores/connect';
-import moment from 'moment';
-import { TRootStore } from 'Stores/index';
+import { observer, useStore } from '@deriv/stores';
 
 type TProgressSliderStream = {
     contract_info: Required<TContractInfo>;
-    is_loading: boolean;
-    server_time: moment.Moment;
 };
 
-const ProgressSliderStream = ({ contract_info, is_loading, server_time }: TProgressSliderStream) => {
+const ProgressSliderStream = observer(({ contract_info }: TProgressSliderStream) => {
+    const { common, portfolio } = useStore();
+    const { server_time } = common;
+    const { is_loading } = portfolio;
+
     if (!contract_info) {
         return <div />;
     }
+    if (!server_time) return null;
```

Comment:
@henry-deriv please move this line down after `const current_tick ..` to keep return statements together🙏🏻